src/runner.cc

```diff
@@ -128,7 +127,18 @@ void run(std::vector<std::string> sql_files, Config config) {
 }  // namespace
 }  // namespace zetasql::linter
 
+class B {
```

Comment:
Accidental debug stuff?

Reply:
Yes, it was for testing `std::vector<std::unique_pointer<bool>> non_used_variable;` when I added this to LinterOptions I got errors, and I think I can't convert to it.

I erased it, but then it returned when I try to recover stuff with ctrl-z (didn't realized that)

Reply:
it is fixed now.

Reply:
ahm, you brought it back in the last commit :D

Reply:
Oh, it was back at 'unique_ptr' commit, but erased again in the last one 'review changes'. So, it is fine now.